notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb

```diff
@@ -99,7 +99,7 @@
       "source": [
```

Comment:
<p>It's a mouthful but I think the most unambiguous terminology would be </p><p> </p><p>(Option A) "… or Vertex AI Workbench user-managed notebooks"</p><p> </p><p>—and actually I think just the following would make sense:</p><p> </p><p>(Option B) "…&nbsp;or Vertex AI Workbench notebooks"</p><p> </p><p>(the implication being that the authentication/setup is taken care of for you in user-managed notebooks AND managed notebooks; I think this is true, but you know more about managed notebooks than me)</p><p> </p><p>Could you do another find and replace to one of my two suggestions, depending on which you think makes sense?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I'll go with "or Vertex AI Workbench notebooks"</p>

Reply:
Done